src/tests/fixtures.py

```diff
@@ -98,3 +101,21 @@ def get_event_message_fixture(event_name=None):
     message['header'] = metadata
 
     return message
+
+
+def get_event_data_fixure():
+    event = {
+        'version': '0.1',
+        'catalogue': 'test_catalogue',
+        'application': 'TEST',
+        'entity': 'test_entity',
+        'timestamp': datetime.datetime(2019, 1, 30, 18, 7, 7),
+        'source': 'test'
+    }
+
+    data = {'_last_event': 1, '_source_id': 'id 2', '_entity_source_id': 'id 2'}
+    return event, data
```

Comment:
if you add action to the event and convert it toma namedtuple here then you don’t need dict to object

Reply:
data can be an empty object

Reply:
data = {'_last_event': 1}

_last_event is used in tests